src/WebJobs.Script.WebHost/WebJobsScriptHostService.cs

```diff
@@ -361,6 +364,8 @@ private async Task UnsynchronizedStartHostAsync(ScriptHostStartupOperation activ
                     deferredLogProvider.ProcessBufferedLogs(selectedProviders);
                 }
 
+                _hostBuiltChangeTokenSourceForLanguageWorkerOptions.TriggerChange();
```

Comment:
This really only works for the `LanguageWorkerOptions`. i.e. If I wanted to have an `IOptionsMonitor<SomeOtherOptions>` that re-built whenever the host was built, this wouldn't trigger. We likely need the `IChangeToken` to be static here and wrapped in the `HostBuiltChangeTokenSource<TOptions>`. We do this today with `StandbyManager.ChangeToken` -- triggering that one static token fires the event for anyone wanting to know when we've specialized.

 It's probably okay for now... but a little refactoring could make it usable elsewhere too.

Reply:
Yea, I see your point. Let me follow up offline,